frontend/app/content/navbar.ts

```diff
@@ -0,0 +1,48 @@
+// https://ui.nuxt.com/components/navigation-menu#usage
+
+export const NAVBAR_ITEMS = [
+  [
+    {
+      label: 'Ranking',
+      to: '/leaderboard',
+    },
+    {
+      label: 'Regulamin',
+      to: '/rules',
+    },
+    {
+      label: 'FAQ',
+      to: '/faq',
+    },
```

Comment:
What about tasks? If you want to move the link to the `/panel` page, create a separate issue for it and leave it for now here